plugins/MobileMessaging/SMSProvider.php

```diff
@@ -37,6 +37,12 @@
 			</li>
 			</ul>
 			',
+		'FreeMobile' => 'You can use your Free Mobile contract to send SMS Reports from Piwik to yourself.<br/>
```

Comment:
I think it should be something like:

You can use your Free Mobile contract to send SMS Reports from Piwik.
* First, activate option in your Free Mobile account settings ? (I'm not sure where something needs to be activated, would be good to be as specific as possible, maybe even post a link if it is the free mobile account)
* Enter your FreeMobile username and password as API Key separated by a whitespace

About Free Mobile
* Free Mobile can be SMS only to the contract owner
* Free Mobile is a French Network Operator